src/renderer/coremods/settings/pages/QuickCSS.tsx

```diff
@@ -22,6 +28,38 @@ type ThemeModule = {
   removeChangeListener: (listener: () => unknown) => unknown;
 };
 
+const PopoutModule = await webpack.waitForModule(
+  webpack.filters.bySource('type:"POPOUT_WINDOW_OPEN"'),
+);
+// eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+const openPopout = webpack.getFunctionBySource(PopoutModule, "POPOUT_WINDOW_OPEN") as (
+  key: string,
+  render: ReactComponent<unknown>,
+  features: Record<string, string>,
+) => void;
+// eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+const closePopout = webpack.getFunctionBySource(PopoutModule, "POPOUT_WINDOW_CLOSE") as (
+  key: string,
+) => void;
+
+// eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+const setAlwaysOnTop = webpack.getFunctionBySource(
+  PopoutModule,
+  "POPOUT_WINDOW_SET_ALWAYS_ON_TOP",
+) as (key: string, alwaysOnTop: boolean) => void;
+
+// eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+const PopoutWindowStore = webpack.getByStoreName("PopoutWindowStore") as Store & {
+  getWindow: (key: string) => Window;
+  getWindowOpen: (key: string) => boolean;
+  getIsAlwaysOnTop: (key: string) => boolean;
+};
+
+// eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+const DnDProvider = webpack.getBySource(
```

Comment:
To be precise, do not confuse the two components, which are different (DragAndDropProvider vs PopoutWindow)
```suggestion
const PopoutWindow = webpack.getBySource(
```